src/logic/map_objects/tribes/productionsite.cc

```diff
@@ -427,7 +425,8 @@ void ProductionSite::format_statistics_string() {
 	// asked for the statistics string. However this string should only then be constructed.
 
 	// boost::format would treat uint8_t as char
-	const unsigned int percent = get_actual_statistics();
+	const unsigned int percent =
+	   (get_actual_statistics() * 100 / 98 > 100) ? 100 : get_actual_statistics() * 100 / 98;
 
```

Comment:
std::min(get_actual_statistics() * 100 / 98, 100)
?

Reply:
should be more efficient. Thanks